llvm/include/llvm/IR/IntrinsicsAssigner.td

```diff
@@ -28,3 +28,17 @@ def int_assigner_bit_composition: Intrinsic<[llvm_any_ty], [llvm_ptr_ty, llvm_i3
 def int_assigner_exit_check: Intrinsic<[], [llvm_i1_ty]>;
 
 def int_assigner_print_native_pallas_field: Intrinsic<[], [llvm_any_ty]>;
+
+def int_assigner_fri_lin_inter: Intrinsic<[llvm_any_ty], [LLVMMatchType<0>, LLVMMatchType<0>, LLVMMatchType<0>, LLVMMatchType<0>]>;
+def int_assigner_fri_cosets: Intrinsic<[], [llvm_ptr_ty, llvm_i32_ty, llvm_any_ty, LLVMMatchType<0>]>;
+def int_assigner_fri_array_swap: Intrinsic<[], [llvm_ptr_ty, llvm_i32_ty, llvm_ptr_ty, llvm_i1_ty]>;
+def int_assigner_gate_arg_verifier: Intrinsic<[llvm_any_ty], [llvm_ptr_ty, llvm_ptr_ty, llvm_i32_ty, llvm_ptr_ty, llvm_i32_ty, LLVMMatchType<0>]>;
+def int_assigner_permutation_arg_verifier: Intrinsic<[llvm_any_ty], [llvm_ptr_ty, llvm_ptr_ty, llvm_ptr_ty, llvm_i32_ty, llvm_any_ty, LLVMMatchType<1>, LLVMMatchType<1>, LLVMMatchType<1>, LLVMMatchType<1>, llvm_any_ty]>;
```

Comment:
I would suggest to use `llvm_anyvector_ty`where it's applicable, just to make the code more readable